src/layouts/RabitLayout/Sidebar/SidebarMenu/items.ts

```diff
@@ -23,6 +23,11 @@ const menuItems: MenuItems[] = [
         name: 'Explore',
         link: '/plots',
         icon: AccessTimeIcon
+      },
+      {
+        name: 'Upload',
+        link: '/upload',
+        icon: UploadIcon
```

Comment:
I think it makes more sense to have the Upload button at the top of the list, and then have Explore followed by anything else right under it.

After all, Uploading is primarily what researchers coming to use the application will want to do. If they want to view something instead, odds are they would have received a share link from someone else, and go to it directly

Reply:
fair enough I guess, done